src/middlewares/validarToken.ts

```
@@ -1,50 +1,26 @@
+import { AuthenticationError, decodeToken, getEncodedSecret, getToken } from "@/utils/token";
+import { captureException } from "@sentry/nextjs";
 
 interface ExtendedNextRequest extends NextRequest {
 	user?: any;
 }
 
-const secret = new TextEncoder().encode(process.env.NEXTAUTH_SECRET);
+export const validarTokenMiddleware = async (req : ExtendedNextRequest) => {
+    try {
+        const secret = getEncodedSecret();
+        if(!secret) return Response.json({ message: 'Erro ao verificar token.', detail: 'Secret não fornecido.' }), { status: 500 };
```

Biome (1.9.4)

[error] The comma operator is disallowed.

Its use is often confusing and obscures side effects.

(lint/style/noCommaOperator)